src/components/EntityPage/File/index.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { injectState } from 'freactal';
+import { withTheme } from 'emotion-theming';
+
+import styled from 'react-emotion';
+import Row from '../../../uikit/Row';
+import Column from '../../../uikit/Column';
+
+import { withApi } from 'services/api';
+import {
+  EntityTitleBar,
+  EntityTitle,
+  EntityActionBar,
+  EntityContent,
+  EntityContentSection,
+  EntityContentDivider,
+} from '../';
+
+const enhance = compose(
+  injectState,
+  withTheme,
+  withApi,
+);
+
+const Container = styled(Column)`
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+`;
+
+const FileEntity = ({ state, effects, theme, fileId, ...props }) => {
+  return (
```

Comment:
should we use the react-grid-system here? I feel like here's a page we can benefit from some responsive grid :P

Reply:
Maybe within each section? The entity pages are split into sections with headers then dividing lines. Would be good to put content within them into grid system. But the page itself is just a column of sections

Reply:
👍